src/world/partner/goombario.c

```diff
@@ -74,7 +74,7 @@ s32 func_802BD1D0_3170F0(Evt* script, s32 isInitialCall) {
         D_8010C954 = 0;
     }
 
-    playerData->unk_2F4[1]++;
+    playerData->partnerUsedTime[1]++;
```

Comment:
for this array, are the indexes partner IDs? if so, we should use the enum here instead of just [1] (applies for all world/partner/*.c